setup.cfg

```diff
@@ -16,13 +16,16 @@ markers =
 
 [flake8]
 exclude =
+    .eggs/
     doc/conf.py
 
 [coverage:run]
 parallel = true
 [coverage:report]
 exclude_lines =
+    pragma: no cover
```

Comment:
Could you briefly explain why we need this?

Reply:
This excludes this code
https://github.com/ESMValGroup/ESMValCore/blob/d91371ca97e786f088a978b55e907415079ad1cb/esmvalcore/_recipe/from_datasets.py#L15-L16
from coverage reporting. The import needs to be done like this to be able to use type hints while avoiding circular imports. You can read more about it [here](https://coverage.readthedocs.io/en/7.0.0/excluding.html#excluding-code-from-coverage-py).

Reply:
thanks, was about to ask the same :laughing: